UPGRADE-1.3.md

```diff
@@ -1 +1,27 @@
 # UPGRADE FROM `v1.2.X` TO `v1.3.0`
+
+## Configuration
+
+* **Sylius 1.3** requires **PHP 7.2**, then following configuratio should be changed:
+    * `composer.json`:
+        * `"php": "^7.1"` -> `"php": "^7.2"`
+        * `"config.platform.php": 7.1.18` -> `"config.platform.php": 7.2.4`
+    * `.travis.yml` (if you're using TravisCI):
+        * leave only configuration for `PHP 7.2`
+
+* Copy `vendor/sylius/sylius/env.dist` to your application root:
+    * basing on this file create env files for your environments (`.env`, `.env.test` etc.)
+    * Add env files to `.gitignore`
```

Comment:
Which ones exactly? Having a list to copy-paste would be 11/10

Reply:
Well, all not-dist files 😄 it depends which one you create. We operate on a example of 2 files, `.env.` and `.env.test`, I'd put them in the list

Reply:
What about:
```
basing on this file create env files for your environments (`.env`, `.env.test` etc.) and add those to your `.gitignore` file
```